src/urllib3/response.py

```diff
@@ -118,6 +138,9 @@ def _get_decoder(mode):
     if mode == 'gzip':
```

Comment:
Let's do a if and elif chain here for better readability.

Reply:
There are competing schools of thought here: one is about having "if-elif-elif-else" wherever you can, the other is about "if your "if" ends with a return, there is no need to add "else" and an extra level of indentation.

I felt like the latter was applied in this section of the code, but I'm happy to change it if you like.